web/packages/shared/components/FieldCheckbox/FieldCheckbox.test.tsx

```diff
@@ -52,7 +52,7 @@ test('controlled flow', async () => {
 test('uncontrolled flow', async () => {
   let checkboxRef;
   function TestForm() {
-    const cbRefInternal = useRef();
+    const cbRefInternal = useRef(undefined);
```

Comment:
should this be null? there's actually still a few more files that uses `undefined` but `null` for others but wasn't sure if there was a reason for it

Reply:
This can be undefined, see https://github.com/gravitational/teleport/pull/54460#discussion_r2081237132.